openmeetings-web/src/main/java/org/apache/openmeetings/web/admin/connection/dto/ConnectionListKStreamItem.java

```diff
@@ -0,0 +1,103 @@
+/*
```

Comment:
I see no value in this class :(

Reply:
Did you read my comments on the PR description why I had to introduce this class ?